### DISCUSSION TITLE
Run a Mod in User context which requires system context

### DISCUSSION THREAD

**Discussion by thisispr** [Romanitho/Winget-AutoUpdate#601] [MODs]
hi @Romanitho  @KnifMelti 

how would you do that?

The problem is, that the application requires elevated privileges, which the user does not have. I want to uninstall the app `Microsoft Visual C++ 2015-2022 Redistributable (x86)` during a mod in system context, and then update/install it with user context.

any suggestion?

**Comment by DWGSascha** [comment]
Same Issue here. With Updating Microsoft.DevHome and Teams Classic.

**Comment by Romanitho** [comment]
Not easy to switch between contexts. If the app is detected in User context, WAU runs in that context (and mods too). You'll need to cook something to get this to work, I guess

**Comment by rebootitltd** [comment]
anyone find a good solution to this. i want to deploy this to my managed clients but its not gonna work if they all get popups for elevation
